daq/host.py

```diff
@@ -147,6 +149,10 @@ def _test_enabled(self, test):
         test_module = self._loaded_config['modules'].get(test)
         return test in self._CORE_TESTS or test_module and test_module.get('enabled', True)
 
+    def _get_test_timeout(self, test):
+        test_module = self._loaded_config['modules'].get(test)
+        return test_module.get('timeout', self._DEFAULT_TIMEOUT) if test_module else None
```

Comment:
what are the cases where test is not defined in modules?  Seems like that might be the implicit modules (pass/fail/etc...)... just want to make sure the logic here is correct. Alternative, of course, would be to apply the default timeout in that case as well. Is there a reason not to do that?

The only one I can think of is the "test_hold" module, which is explicitly design/intended to block/hold forever.